ctapipe/io/tableloader.py

```diff
@@ -378,6 +380,8 @@ def read_subarray_events_chunked(self, chunk_size):
             self.read_subarray_events,
             n_total=len(self),
             chunk_size=chunk_size,
+            args=args,
+            kwargs=kwargs,
```

Comment:
Why explicit and not `*args` and `**kwargs`?

Reply:
Because you then have to mix args to the called function with kwargs to the ChunkIterator as you cannot give args after kwargs.